### DISCUSSION TITLE
Unable to manually trigger new runs

### DISCUSSION THREAD

**Discussion by altmas5** [codemagic-ci-cd/codemagic-docs#2107] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Android (Kotlin)

### Steps to reproduce

1. Click on the Start new build button
2. Wait for the list of branches to be pulled

### Expected results

Get the list of branches and workflows to later start the build

### Actual results

The load never ends
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/1054618/15809226-39dd-4e4e-9602-2239c322dc93)

**Comment by zmarakjanbangash** [comment]
Yes same for me. Also happening with tags. Work load the workflows but i do see list of branches and tags

**Reply by dabauum221** [reply]
Did you get it to work?

**Comment by Gaurav-CareMonitor** [comment]
Same here

**Reply by dabauum221** [reply]
Did you get it to work?

**Reply by Gaurav-CareMonitor** [reply]
rather than building manually , I added build on push, so the build is working when pushing changes from GitHub.

**Reply by dabauum221** [reply]
Thanks, easy enough, works for me too

**Comment by dabauum221** [comment]
Same issue for me, so frustrating

**Comment by malvinpratama** [comment]
I test inspect element & I get the build id
but not show in build page
this my build id "6527720706762aa8ffcb99bf" if you want check

**Comment by selva-dcx** [comment]
Same issue here as well

**Comment by altairdeark** [comment]
Same

**Comment by pdichone** [comment]
Same here :(

**Comment by anilchalissery** [comment]
having the same issue, not able to start a new build, branch is listed fine however codemagic.yaml is not loaded

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi everyone, we had an infrastructure incident that lasted from 00:45 UTC to 06:10 UTC. During that time, some users experienced issues with loading branches or codemagic.yaml, starting builds, and adding new applications. The problem affected public repositories and repositories using SSH keys for authentication. OAuth repositories were not affected. We apologize for any inconvenience caused.

**Reply by dabauum221** [reply]
I had quite a few builds kickoff overnight due to this issue.  All these failed of course due to trying to publish to Apple using the same build number.  I also went from 100 minutes used to 217 minutes used this month.  Can these be fixed?  Can anything that happened during this time on my account be wiped and my Codemagic next build number be reset?

**Reply by Gaurav-CareMonitor** [reply]
Same, We also have 10+ builds kickoff due to this issue.
